packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -92,11 +93,12 @@ export const ChatList = ({
   onMessageReceived,
   onAllMessagesRead,
   onLoaded,
+  onConnectionChanged,
   chatThreadsPerPage,
   ...props
 }: MgtTemplateProps & IChatListItemProps & IChatListMenuItemsProps) => {
```

Comment:
we should change `IChatListItemProps` to `IChatListProps`

Reply:
And in ChatListItem, which you didn't touch, change to `IChatListItemProps`

Reply:
done